recipients_app/lib/my_app.dart

```diff
@@ -42,6 +42,11 @@ class MyApp extends StatelessWidget {
             firestore: firestore,
           ),
         ),
+        RepositoryProvider(
+          create: (context) => SurveyRepository(
```

Comment:
Okay, SurveyCubit is not on top of the MaterialApp, sorry thought it was. Do you think it would make sense? Otherwise just use it where you need it like described in the comments

Reply:
We need `recipient` probably, so it is similar to payments cubit. Not sure if we can make it top level